bridge-sdk/bridge-clients/evm-bridge-client/src/evm_bridge_client.rs

```diff
@@ -202,11 +212,12 @@ impl EvmBridgeClient {
             }
         }
 
-        let value = if token == H160::zero() {
-            U256::from(fee.native_fee.0) + U256::from(amount)
-        } else {
-            U256::from(fee.native_fee.0)
-        };
+        let wormhole_fee = wormhole_core.message_fee().call().await?;
```

Comment:
We don't use wormhole for mainnet so this will be wasteful. Let's think how we can call it only for non-mainnet chains.

My first thought is to add `ChainKind` field to the client and decide based on that? Then we should also remove `chain_id` and determine it based on `ChainKind`